Sources/MapboxMaps/Gestures/GestureOptions.swift

```diff
@@ -73,5 +73,8 @@ public struct GestureOptions: Equatable {
     /// Defaults to `UIScrollView.DecelerationRate.normal.rawValue`
     public var panDecelerationFactor: CGFloat = UIScrollView.DecelerationRate.normal.rawValue
 
+    /// By default, gestures rotate and zoom around the center of the gesture. Set this property to rotate and zoom around a fixed point instead.
```

Comment:
Let's assume that https://github.com/mapbox/mapbox-maps-ios/pull/1092 will land for the beta and update this comment to encourage setting pinchPanEnabled to false when using a non-nil focalPoint.

Reply:
Good point, added a sentence about it.